contributors/devel/on-call-build-cop.md

```diff
@@ -31,7 +31,7 @@ longer a part of the mandate.
 ## Workflow
 
 1. Check the Prow batch dashboard: [https://prow.k8s.io/?type=batch](https://prow.k8s.io/?type=batch) 
-to ensure that merges are occurring regularly.
+to ensure that batch jobs are running regularly. It's okay to see occasional flakes.
```

Comment:
I would keep the original text.  The buildcop actually cares about the merge rate, not that the batch jobs are running (seems like that would be a test-infra-oncall problem, no?).